src/components/MetaData/index.tsx

```diff
@@ -69,14 +81,26 @@ interface Props {
     };
   } | null;
   websiteName?: string | null;
+  websiteDomain?: string | null;
   twitterUsername?: string | null;
   resizerURL?: string | null;
+  arcSite?: string | null;
+  facebookAdmins?: string | null;
 }
 
 const MetaData: React.FC<Props> = ({
-  MetaTag, MetaTags, metaValue, globalContent: gc, websiteName, twitterUsername, resizerURL,
+  MetaTag,
+  MetaTags,
+  metaValue,
+  globalContent: gc,
+  websiteName,
+  websiteDomain,
+  twitterUsername,
+  resizerURL,
+  arcSite,
+  facebookAdmins,
 }) => {
-  const pageType = metaValue('page-type') || '';
+  const pageType = metaValue('page-type');
```

Comment:
what's the expected behavior if there is an undefined `pageType`?

Reply:
good catch! that condition isn't tested.

Reply:
looks like this is resolved via test